tests/unit/cli_test.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import sys
+
+import mock
+import pytest
+
+import pgctl.cli
+from pgctl.cli import _humanize_seconds
+from pgctl.cli import PgctlApp
+from pgctl.cli import TermStyle
+from pgctl.daemontools import SvStat
+
+
+@pytest.mark.parametrize(('seconds', 'expected'), [
+    (0, '0 seconds'),
+    (59, '59 seconds'),
+    (60, '1.0 minutes'),
+    (7000, '1.9 hours'),
+])
+def test_humanize_seconds(seconds, expected):
+    assert _humanize_seconds(seconds) == expected
+
+
+def test_termstyle_wrap_no_colors_if_no_tty():
+    with mock.patch.object(sys.stdout, 'isatty', return_value=False):
+        assert TermStyle.wrap('hello world', TermStyle.RED) == 'hello world'
+
+
+def test_termstyle_wrap_uses_colors_if_tty():
+    with mock.patch.object(sys.stdout, 'isatty', return_value=True):
+        assert (
+            TermStyle.wrap('hello world', TermStyle.RED) ==
+            '\033[91mhello world\033[0m'
+        )
+
+
+def fake_statuses(statuses):
+    app = PgctlApp()
+    app.services = []
+    for name, status in statuses:
+        m = mock.Mock(**{'svstat.return_value': status})
+        m.name = name
```

Comment:
mocking out `name` is the worst :\

almost impossible to write this as a generator expression because of that